frontend/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "frontend",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "@testing-library/jest-dom": "^5.11.10",
+    "@testing-library/react": "^11.2.6",
+    "@testing-library/user-event": "^12.8.3",
+    "react": "^16.0.0",
```

Comment:
sorry I pranked us. Might be easier if we use the same libraries that Andrew is using for assignment 1. Should just be a case of editing this file to include the following:
```
    "@material-ui/core": "^4.11.3",
    "@material-ui/icons": "^4.11.2",
    "@material-ui/pickers": "^3.3.10",
    "axios": "^0.21.1",
```
(under dependencies)
and
```
    "@types/jest": "^26.0.22",
    "@wojtekmaj/enzyme-adapter-react-17": "^0.6.0",
    "enzyme": "^3.11.0",
    "jest-enzyme": "^7.1.2",
    "jest-mock-axios": "^4.3.0",
    "react-test-renderer": "^17.0.2"
```
(under devDependencies)

Reply:
There are a bunch of other dependencies that he has (i.e. workbox) but maybe we should only install those if/when we know we are going to use them

Reply:
Done! I have added these and restored the original react version :) thanks for this!